geth/utils/validation.py

```diff
@@ -86,12 +58,12 @@ class GethKwargs(BaseModel):
     ws_api: str | None = None
     ws_enabled: bool | None = None
     ws_origins: str | None = None
-    ws_port: int | None = None
+    ws_port: str | None = None
```

Comment:
Do you know if this was expected to be a `str` or an `int` before? I think `str` makes good enough sense.

Reply:
I changed this based on what I found in `wrapper.py` in setting the defaults, and that the other keys `port` and `rpc_port` were strings.